rdflib/namespace.py

```diff
@@ -246,11 +246,69 @@ def term(self, name):
 
 XSD = Namespace(_XSD_PFX)
 
-SKOS = Namespace('http://www.w3.org/2004/02/skos/core#')
-DOAP = Namespace('http://usefulinc.com/ns/doap#')
-FOAF = Namespace('http://xmlns.com/foaf/0.1/')
+CSVW = Namespace('http://www.w3.org/ns/csvw#')
 DC = Namespace('http://purl.org/dc/elements/1.1/')
+DCAT = Namespace('http://www.w3.org/ns/dcat#')
 DCTERMS = Namespace('http://purl.org/dc/terms/')
+DOAP = Namespace('http://usefulinc.com/ns/doap#')
+FOAF = ClosedNamespace(
+    uri=URIRef('http://xmlns.com/foaf/0.1/'),
+    terms=[
+        # all taken from http://xmlns.com/foaf/spec/
+        'Agent', 'Person', 'name', 'title', 'img',
+        'depiction', 'depicts', 'familyName',
+        'givenName', 'knows', 'based_near', 'age', 'made',
+        'maker', 'primaryTopic', 'primaryTopicOf', 'Project', 'Organization',
+        'Group', 'member', 'Document', 'Image', 'nick',
+        'mbox', 'homepage', 'weblog', 'openid', 'jabberID',
+        'mbox_sha1sum', 'interest', 'topic_interest', 'topic', 'page',
+        'workplaceHomepage', 'workInfoHomepage', 'schoolHomepage', 'publications', 'currentProject',
+        'pastProject', 'account', 'OnlineAccount', 'accountName', 'accountServiceHomepage',
+        'PersonalProfileDocument', 'tipjar', 'sha1', 'thumbnail', 'logo'
```

Comment:
The Travis CI errors appears to be complaining about the missing `firstName`.

![image](https://user-images.githubusercontent.com/304786/76055598-f79c1f00-5fd8-11ea-8a8b-c754a12c4a60.png)

![image](https://user-images.githubusercontent.com/304786/76055621-0551a480-5fd9-11ea-8152-3831e3052e01.png)

Reply:
A `doctest` is referencing that field too, which caused the Travis failure.

https://github.com/RDFLib/rdflib/blob/51775c07793b6edfaa8d89c4f15c9dbb91c87858/rdflib/extras/describer.py#L52

Reply:
Yep, just word that out and have just updated.